### ISSUE TITLE
Error in zmq interface

### ISSUE THREAD

**Issue by adeas31** [OpenModelica/OMPython#130]
Only happens when you have some error message. Is probably because of my swedish locale.

```
Python 3.8.5 (tags/v3.8.5:580fbb0, Jul 20 2020, 15:57:54) [MSC v.1924 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> from OMPython import OMCSessionZMQ
>>> omc = OMCSessionZMQ()
>>> omc.sendExpression("loadModel(Modelica)")
True
>>> omc.sendExpression("getNthInheritedClass(Modelica.Fluid.Machines.Pump, 1)")
'Modelica.Fluid.Machines.BaseClasses.PartialPump'
>>> omc.sendExpression("getErrorString()")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\OpenModelica\OMWebService\myenv\lib\site-packages\OMPython\__init__.py", line 776, in sendExpression
    result = self._omc.recv_string()
  File "C:\OpenModelica\OMWebService\myenv\lib\site-packages\zmq\sugar\socket.py", line 684, in recv_string
    return self._deserialize(msg, lambda buf: buf.decode(encoding))
  File "C:\OpenModelica\OMWebService\myenv\lib\site-packages\zmq\sugar\socket.py", line 590, in _deserialize
    return load(recvd)
  File "C:\OpenModelica\OMWebService\myenv\lib\site-packages\zmq\sugar\socket.py", line 684, in <lambda>
    return self._deserialize(msg, lambda buf: buf.decode(encoding))
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xe4 in position 108: invalid continuation byte
>>>
```

**Comment by arun3688** [comment]
should this work?  `omc.sendExpression("getErrorString()").encode().decode()`

**Comment by bilderbuchi** [comment]
Is whatever `getErrorString()` returns encoded in utf-8 (and not e.g. this Windows-specific encoding)? 
Is there a "J**ä**vla" in the error ;-)? (0xe4 seems to be "ä")

**Comment by adeas31** [comment]
> should this work? `omc.sendExpression("getErrorString()").encode().decode()`

This didn't work.

**Comment by sjoelund** [comment]
You'll need to fix OMC's gettext to return UTF-8 text instead of Windows-1252 I guess. All strings should be Unicode...

**Comment by sjoelund** [comment]
Possibly, add a check in the OMC ZMQ interface that all strings are Unicode

**Comment by adeas31** [comment]
Fixed with #131
